kettle-plugins/hbase/src/main/java/org/pentaho/big/data/kettle/plugins/hbase/input/HBaseInputDialog.java

```diff
@@ -1257,10 +1252,21 @@ public HBaseConnection getHBaseConnection() throws IOException, ClusterInitializ
       defaultConf = transMeta.environmentSubstitute( m_defaultConfigText.getText() );
     }
 
-    if ( Const.isEmpty( zookeeperHosts ) && Const.isEmpty( coreConf ) && Const.isEmpty( defaultConf ) ) {
-      throw new IOException( BaseMessages.getString( HBaseInputMeta.PKG,
-        "MappingDialog.Error.Message.CantConnectNoConnectionDetailsProvided" ) );
-    }
+    //    NamedCluster nc = namedClusterWidget.getSelectedNamedCluster();
```

Comment:
Please remove these commented code if it is not being used

Reply:
Instead of commenting the code, I have added a check to find whether this flow is for Knox or not.